bot/handlers/sending_videos/send_clip_handler.py

```diff
@@ -23,39 +27,56 @@ class SendClipHandler(BotMessageHandler):
     def get_commands(self) -> List[str]:
         return ["wyślij", "wyslij", "send", "wys"]
 
+    def _get_validator_functions(self) -> ValidatorFunctions:
+        return [
+            self.__check_argument_count,
+        ]
+
+    async def __check_argument_count(self, message: Message) -> bool:
+        return await self._validate_argument_count(message, 2, get_give_clip_name_message())
+
     async def _do_handle(self, message: Message) -> None:
         content = message.text.split()
-        if len(content) < 2:
-            return await self._reply_invalid_args_count(message, "📄 Podaj nazwę klipu. Przykład: /wyślij nazwa_klipu")
+        clip_number = None
+        clip_identifier = " ".join(content[1:])
+
+        if clip_identifier.isdigit():
+            clip_number = int(clip_identifier)
+            clips = await DatabaseManager.get_saved_clips(message.from_user.id)
+            if clip_number < 1 or clip_number > len(clips):
```

Comment:
```suggestion
            if clip_number not in range(1, len(clips)):
```

chyba bardziej pytongowo ale nie wiem

Reply:
Fakt, za dużo C++/ASM mam w głowie XD